vortexasdk/api/eia_forecast.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from typing import List
+
+from vortexasdk.api.serdes import FromDictMixin
+from vortexasdk.api.shared_types import IDName
+
+
+@dataclass(frozen=True)
+class EIAForecast(IDName, FromDictMixin):
```

Comment:
This `EIAForecast` class currently inherits from IDName - which should only be the case if each EIAForecast record returns by the API has both an `id` attribute and a `name` attribute. @dufia This is likely the cause of the failing tests

![Uploading image.png…]()

Reply:
solved 👍 thanks